<?xml version="1.0" encoding="utf-8"?>
<launch>
  <node name = "pub_mono_cam" pkg = "cam" type = "pub_mono_cam"/>
  <node name = "obstacle_detection" pkg = "cam" type = "obstacle_detection" output="screen"/>
</launch>
